util/unifyfs/src/unifyfs.c

```diff
@@ -258,9 +265,47 @@ int main(int argc, char** argv)
         if (NULL == cli_args.share_dir) {
             printf("USAGE ERROR: shared directory (-S) is required!\n");
             usage(1);
+            return -EINVAL;
         }
+        if (cli_args.stage_in != NULL) {
+            if (cli_args.script) {
+	        fprintf(stderr,
+			"WARNING! You are using a script with --stage-in.\n");
+		fprintf(stderr, "This is won't work.\n");
+		return -EINVAL;
+            }
+            if (stat(cli_args.stage_in, &manifest_file_stat)) {
+	        fprintf(stderr,
+			"Cannot read stagein manifest file:%s\n",
+			cli_args.stage_in);
+		return -ENOENT;
+            }
+	}
         return unifyfs_start_servers(&resource, &cli_args);
     } else if (action == ACT_TERMINATE) {
+        if (cli_args.stage_out != NULL) {
+	    // status directory isn't required just to terminate the servers
+	    // but it IS required if we're calling for stage-out, so that
+	    // the stage-out can be started, then the server terminate waits
+	    // until the stage-out is done.
+            if (NULL == cli_args.share_dir) {
+	        printf("USAGE ERROR: shared directory (-S) is required!\n");
+		usage(1);
+		return -EINVAL;
+            }
+            if (cli_args.script) {
+	        fprintf(stderr,
+			"WARNING! You are using a script with --stage-out.\n");
+		fprintf(stderr, "This won't work.\n");
+		return -EINVAL;
+            }
+            if (stat(cli_args.stage_out, &manifest_file_stat)) {
+	        fprintf(stderr,
+			"Cannot read stageout manifest file:%s\n",
+			cli_args.stage_out);
+		return -ENOENT;
```

Comment:
Fix your indents

Reply:
I'll run astyle on this and check it in with the next patch.

Reply:
This has been fixed and will be pushed to the branch soon.